src/core/previous.js

```diff
@@ -18,5 +20,7 @@ export default ($ref: HTMLElement, $slides: HTMLElement, store: Store<SliderStat
         }));
     }
 
-    requestAnimationFrame(() => slideTo(store, currentSlide - slideOffset));
+    !options.infinite && currentSlide - slideOffset <= 0 ? _slideTo = 0 : _slideTo = currentSlide - slideOffset;
```

Comment:
Please change this to 

_slideTo = !options.infinite && currentSlide - slideOffset <= 0 ? 0 : currentSlide - slideOffset;